provider/lib.go

```diff
@@ -444,13 +452,22 @@ func flattenRuleRateLimit(rateLimit *sigsci.RateLimit) map[string]string {
 	}
 }
 
-func flattenRuleActions(actions []sigsci.Action) []interface{} {
+func flattenRuleActions(actions []sigsci.Action, customResponseCode bool) []interface{} {
 	var actionsMap = make([]interface{}, len(actions), len(actions))
 	for i, action := range actions {
+
 		actionMap := map[string]interface{}{
 			"type":   action.Type,
 			"signal": action.Signal,
 		}
+		// customResponseCode is enabled for site rules but disabled for corp rules
+		// this boolean flag reflects the differences and flattens objects accordinglyg
+		if customResponseCode {
```

Comment:
i dont think this check will be needed if you put computed:true into corp rule. by doing that you wont be working with two different data structures on the provider side.